opencog/spacetime/octomap/OctoMapNode.cc

```diff
@@ -0,0 +1,64 @@
+/*
+ * opencog/spacetime/octomap/OctoMapNode.cc
+ *
+ * Copyright (C) 2018 Hanson Robotics
+ * All Rights Reserved
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License v3 as
+ * published by the Free Software Foundation and including the
+ * exceptions at http://opencog.org/wiki/Licenses
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with this program; if not, write to:
+ * Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+
+#include <mutex>
+
+#include <opencog/atoms/value/NameServer.h>
```

Comment:
and this one: `opencog/atoms/atom_types/NameServer.h`

Sorry for all the trouble. This really should be the last of it.   We are going to move truth values a bit later, but after that, there will be no more big code reorgs that affect everybody.